src/fava/application.py

```diff
@@ -270,11 +270,19 @@ def fava_api_exception(error: FavaAPIException):
 
 @app.route("/")
 @app.route("/<bfile>/")
-def index():
+def index(bfile=None):
     """Redirect to the Income Statement (of the given or first file)."""
     if not g.beancount_file_slug:
         g.beancount_file_slug = next(iter(app.config["LEDGERS"]))
-    return redirect(url_for("report", report_name="income_statement"))
+    default_path = ""
+    if not bfile:
+        default_path += g.beancount_file_slug + "/"
+    default_path += (
+        app.config["LEDGERS"][g.beancount_file_slug]
+        .fava_options["default-path"]
+        .lstrip("/")
```

Comment:
Remove this, let's make this option a bit stricter by having to specify a relative path.

Reply:
To be clear, you're asking me to remove only `.lstrip("/")`, correct?

Reply:
Yes.

Reply:
I've done this and updated the documentation to reflect that it is required to be a relative path.